<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ns::SettingsBlacklistPage</class>
 <widget class="QWidget" name="ns::SettingsBlacklistPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>409</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Blacklist</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../translator.qrc">
    <normaloff>:/images/ui/act_blacklist.png</normaloff>:/images/ui/act_blacklist.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-family: Verdana, &quot;宋体&quot;, Arial, Sans-serif, Segoe UI, Tahoma;
}

#widget {
	background: #F2F9FF;
}

#frameBlacklistLine {
	background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0,stop: 0 #80ACCF, stop: 1.0 #F2F9FF);
}

#lineEditUser {
	border: 1px solid #9CB1C2;
	border-radius: 3px;
}

#pushButtonAdd {
	color: #FFFFFF;
}

/* Tree Widget */
#treeWidgetBlacklist {
	border: 1px solid #BDD8EB;
	border-radius: 3px;
	show-decoration-selected: 1;
}

#treeWidgetBlacklist::item {
	padding: 3px;
	border-top-color: transparent;
	border-bottom-color: transparent;
	border-bottom: 1px solid #E6F3FC;
	height: 34px;
}

#treeWidgetBlacklist::item:hover {
	background: #F2F9FF;
}

#treeWidgetBlacklist QHeaderView::section {
	border: 0px;
	background-color: #E7F4FD;
	padding-left: 9px;
    border-bottom: 1px solid #D9ECFB;
	height: 28px;
}

#treeWidgetBlacklist QScrollBar:vertical {
	border-image: url(:/images/ui/bg_scroll_bar.png);
	width: 14px;
	margin: 20px 0 20px 0;
	padding: 0 2px 0 2px;
}

#treeWidgetBlacklist QScrollBar::handle:vertical {
	border-image: url(:/images/ui/scroll_handle.png) 4 0 4 0;
	min-height: 18px;
	border-top: 4px transparent;
	border-right: 0px transparent;
	border-bottom: 4px transparent;
	border-left: 0px transparent;
}

#treeWidgetBlacklist QScrollBar::handle:vertical:hover {
	border-image: url(:/images/ui/scroll_handle_hover.png) 4 0 4 0;
}

#treeWidgetBlacklist QScrollBar::handle:vertical:pressed {
	border-image: url(:/images/ui/scroll_handle_pressed.png) 4 0 4 0;
}

#treeWidgetBlacklist QScrollBar::add-line:vertical {
	border-left: 1px solid #DDE8ED;
	border-right: 1px solid #DDE8ED;
	background: #F5F7F8;
	height: 20px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}

#treeWidgetBlacklist QScrollBar::sub-line:vertical {
	border-left: 1px solid #DDE8ED;
	border-right: 1px solid #DDE8ED;
	background: #F5F7F8;
	height: 20px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}

#treeWidgetBlacklist QScrollBar::up-arrow:vertical {
    width: 10px;
    height: 5px;
    background: url(:/images/ui/scroll_top_arrow.png);
}

#treeWidgetBlacklist QScrollBar::up-arrow:vertical:hover {
    background: url(:/images/ui/scroll_top_arrow_hover.png);
}

#treeWidgetBlacklist QScrollBar::up-arrow:vertical:pressed {
    background: url(:/images/ui/scroll_top_arrow_pressed.png);
}

#treeWidgetBlacklist QScrollBar::down-arrow:vertical {
	width: 10px;
    height: 5px;
    background: url(:/images/ui/scroll_bottom_arrow.png);
}

#treeWidgetBlacklist QScrollBar::down-arrow:vertical:hover {
    background: url(:/images/ui/scroll_bottom_arrow_hover.png);
}

#treeWidgetBlacklist QScrollBar::down-arrow:vertical:pressed {
    background: url(:/images/ui/scroll_bottom_arrow_pressed.png);
}

#treeWidgetBlacklist QScrollBar::add-page:vertical, QTreeView QScrollBar::sub-page:vertical {
	background: none;
}

#pushButtonAddBlacklist {
	border-radius: 3px;
	color: #FFFFFF;
	background: #2c8bdb;
}

#pushButtonAddBlacklist:hover {
	background: #1f79c5;
}

#pushButtonAddBlacklist:pressed {
	background: #1665a7;
}
/* Tree Widget */</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="leftMargin">
       <number>18</number>
      </property>
      <property name="topMargin">
       <number>18</number>
      </property>
      <property name="rightMargin">
       <number>18</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="labelBlacklistDescription">
          <property name="text">
           <string>Blacklist management</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frameBlacklistLine">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>1</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>30</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>49</number>
         </property>
         <item>
          <widget class="QLabel" name="labelBlacklistTitle">
           <property name="text">
            <string>The user in the blacklist can't use the service</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelAddBlacklist">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Add blacklist</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineEditBlacklistFilter">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>ID</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonAddBlacklist">
             <property name="minimumSize">
              <size>
               <width>53</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>53</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="ns::CustomTreeWidget" name="treeWidgetBlacklist">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>240</height>
            </size>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="indentation">
            <number>0</number>
           </property>
           <attribute name="headerMinimumSectionSize">
            <number>0</number>
           </attribute>
           <attribute name="headerStretchLastSection">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>163</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ns::CustomTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>customtreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../translator.qrc"/>
 </resources>
 <connections/>
</ui>
